src/mpl2/test/guides1.tcl

```diff
@@ -0,0 +1,25 @@
+# Test if the guidance region is moving the macro
+# to the left side (without a guide, the macro ends 
+# up on the right side)
+source "helpers.tcl"
+
+read_lef "./Nangate45/Nangate45.lef"
+
+read_lef "./testcases/macro_only.lef"
+read_liberty "./testcases/macro_only.lib"
+
+read_verilog "./testcases/io_constraints1.v"
+link_design "io_constraints1"
+read_def "./testcases/io_constraints1.def" -floorplan_initialize
+
+#set_io_pin_constraint -direction INPUT -region left:*
+#place_pins -annealing -random -hor_layers metal5 -ver_layer metal6
```

Comment:
Sorry, it looks like the test will error, because mpl2 will be unhappy with unplaced pins. Please, leave the PPL cmd as it was but adapt the guide region to occupy the lower right corner of the core.